_data/cloudservices.yml

```diff
@@ -1907,10 +1907,33 @@ services:
             ref:
             icon: none.png
       - huawei:
-          - name: DevStar
-            ref: https://developer.huaweicloud.com/resource/devstar.html
-            icon: devstar.png
```

Comment:
I see there is no other reference to devstar. 
Can we also remove this not used asset `devstar.png`

Reply:
Done